src/bitgo.js

```diff
@@ -219,6 +223,7 @@ const BitGo = function(params) {
     this._baseUrl = common.Environments[env].uri;
   }
 
+  this._microservicesUrl = common.Environments[env].microservicesUri;
```

Comment:
Couldn't this just be `this._microservicesUrl = params.microservicesUrl`?

Reply:
yes, my reasoning is in the above comment. I'm fine with changing it.